packages/icon/src/css/index.js

```diff
@@ -25,7 +25,9 @@ export default {
     width: vars.widths.large
   },
   [`.psds-icon > svg`]: {
-    fill: 'currentColor'
+    fill: 'currentColor',
+    width: '100%',
+    height: '100%'
```

Comment:
This is optional but I noticed icon wasn't aligning correctly in it's container. We can do this style or we can do the following

```
[`.psds-icon`]: {
    display: 'inline-flex',
   alignItems: 'center',
   justifyContent: 'center,
 }
```